contracts/spec_interfaces/IDelegation.sol

```diff
@@ -4,6 +4,26 @@ import "./IContractRegistry.sol";
 
 /// @title Elections contract interface
 interface IDelegations /* is IStakeChangeNotifier */ {
+	/// @dev Notifies of stake change event.
```

Comment:
it's already inherited, the need to expose in the interface is solved by the change to IStakeChangeNotifier